### PR TITLE
Added Panzer Paladin port.

### PR DESCRIPTION
**Pull request by JohnnyonFlame**
# New Port for Panzer Paladin

## Game Information
- **Title**: Panzer Paladin
- **URL**: https://github.com/JohnnyonFlame/FNAPatches

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [ ] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [ ] 720x720 (RGB30)
- [ ] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/panzerpaladin/port.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
```
Comment (kloptops) [inline ports/panzerpaladin/port.json:2]: This should be: `"version": 2,`

**Review by kloptops — CHANGES_REQUESTED** [review]
Just the wrong version in the port.json

**Comment by JohnnyonFlame** [comment]
I have defeated the workflow.

**Comment by JohnnyonFlame** [comment]
Also fine on Batocera.

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**